### PR TITLE
docs: Document appimage artifacts

### PR DESCRIPTION
**Pull request by danobi**
Document how to download/install the new appimages.

##### Checklist

- [ ] Language changes are updated in `man/adoc/bpftrace.adoc` and if needed in `docs/reference_guide.md`
- [ ] User-visible and non-trivial changes updated in `CHANGELOG.md`
- [ ] The new behaviour is covered by tests

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -159,6 +160,29 @@ To install tools and documentation:
...
+
+bpftrace currently ships appimages in two locations:
+
+* in artifacts on official releases
+* as a CI artifact for every build on master
+
+To download the official release artifacts, see the
+[latest release](https://github.com/iovisor/bpftrace/releases/latest).
+
+To download the bleeding edge appimage, go to the
```
Comment (jordalgo) [inline INSTALL.md:177]:
```suggestion
To download the bleeding edge AppImage, go to the
```

File: INSTALL.md [hunk 2]
```diff
@@ -159,6 +160,29 @@ To install tools and documentation:
...
+[workflow page](https://github.com/iovisor/bpftrace/actions/workflows/binary.yml)
+and select the latest run. You should find an uploaded artifact like below:
+
+<img src="./images/ci_appimage_artifact.png" width="40%" height="40%">
+
+Note that Github will automatically place all build artifacts in a .zip - it's
+out of our control. So remember to unzip it first.
```
Comment (jordalgo) [inline INSTALL.md:184]:
```suggestion
Note that Github will automatically place all build artifacts in a .zip (it's
out of our control) so remember to unzip it first.
```

File: INSTALL.md [hunk 3]
```diff
@@ -159,6 +160,29 @@ To install tools and documentation:
 sudo apk add bpftrace-doc bpftrace-tools bpftrace-tools-doc
 ```
 
+# AppImage install
+
+[AppImages](https://appimage.org/) are a portable way to distribute Linux
+applications. To the user, they are functionally equivalent to statically
+linked binaries.
+
+bpftrace currently ships appimages in two locations:
```
Comment (jordalgo) [inline INSTALL.md:169]:
```suggestion
bpftrace currently ships AppImages in two locations:
```

**Review by jordalgo — APPROVED** [review]
Few nitpicks but LGTM

**Comment by danobi** [comment]
Fixed; thanks for looking

**Pull request merged**